### ISSUE TITLE
A11y_VisualStudioCodeClient_Search_Global Search_ Keyboard:Keyboard Focus moving from search to replace instead of moving to “Match case, match whole word and Use regular expression”

### ISSUE THREAD

**Issue by BhanuPrakashGit** [microsoft/vscode#180113]
**GitHub Tags:**
#A11yMAS; #A11yTCS; #A11ySev2; #Device; #Visual Studio Code Client; #BM-VisualStudioCodeClient-MAC-Oct2022; #WCAG2.4.3; #Keyboard; #MAC; #Focus order;

### Environment and OS details:
Visual Studio Code Insiders
Screenreader: Voiceover

### Repro Steps:

1. Open Visual studio code editor
2. Using keyboard navigation View Menu Bar
3. Select search option
4. Verify that keyboard focus is moving from search to Match case, match whole word and use regular expression or not.

### Actual Result:
Keyboard Focus is moving from search to replace instead of moving to “Match case, match whole word and Use regular expression”.

### Expected Result:
Keyboard Focus should move from search to “Match case, match whole word and Use regular expression”.

### Attachment:
![197940893-1835324d-235c-4737-bfda-5527701aa931](https://user-images.githubusercontent.com/104351566/232457766-af9f0eb8-9355-4809-8524-57ff9841a486.gif)

**Comment by BhanuPrakashGit** [comment]
Reference bug: https://github.com/Microsoft/vscode/issues/164697#issue-1423846882

**Comment by meganrogge** [comment]
https://github.com/microsoft/vscode/issues/159880#issuecomment-1247538835

@VijayaBharathyA I agree with what @roblourens stated at the link above. The experience we provide currently is accessible and was designed with the user in mind.

We would reopen this issue if it were a matter of accessibility, but it is a matter of design decision.

**Comment by BhanuPrakashGit** [comment]
@meganrogge It's not an acceptable design because the focus order is completely wrong.
If the navigation sequence is independent of meaning or operation, then the focus order should align with the flow of the visual design.

Could you please reopen this issue.

**Comment by meganrogge** [comment]
@isidorn what do you think of this?

**Comment by roblourens** [comment]
> If the navigation sequence is independent of meaning or operation, then the focus order should align with the flow of the visual design.

Can you explain this sentence some more? In my mind, the navigation sequence is tied to the operation of the feature, that's why we've made it work this way, to match the user's expectation and the way that they use it.

**Comment by BhanuPrakashGit** [comment]
@meganrogge yes, that's right, focus order should be from 'left to right' and 'top to bottom'.
